projects/fsastorefrontstyles/scss/base/comparison-table.scss

```diff
@@ -93,6 +93,9 @@ cx-fs-comparison-table-container {
         font-size: var(--cx-h3-font-size);
       }
     }
+    .fas {
+      line-height: $headings-line-height;
```

Comment:
Needed to align with other titles, so it creates the illusion they are all in the same row